documentation-site/pages/components/layer.mdx

```diff
@@ -23,6 +24,27 @@ and visibility handling, what layer goes on top. There are many other layer
 related issues that include focus management, hover management, keyboard navigation,
 click events and hotkeys usage.
 
+## LayersManager
+
+In order to start leveraging the layers approach add a `LayersManager` to your application. 
+It need to be added at the root of the application. If you use [`BaseProvider`](https://baseweb.design/components/base-provider/) 
+it will add the `LayersManager` for you. Just make sure you add the `BaseProvider` at the root 
+of your application. 
+
+`LayersManager` adds two sibling containers at the root of your application. Your main 
```

Comment:
How about

> `LayersManager` creates two adjacent divs and appends them to the `document.body`. The main React root will be mounted into the first div. The second div is used to host rendered layers.

terms like "sibling containers" or "main application" might not be clear.

Reply:
Edited, slightly changed since it doesn't append the divs to the body but to the container the app is supposed to be added to.